app/models/manageiq/providers/azure/cloud_manager/event_parser.rb

```diff
@@ -0,0 +1,18 @@
+module ManageIQ::Providers::Azure::CloudManager::EventParser
+  extend ManageIQ::Providers::Azure::EventCatcherMixin
+
+  def self.event_to_hash(event, ems_id)
```

Comment:
@blomquisg Is this supposed to be at the class level?  Or should we instantiate a parser?